pkg/mixin/provider/install_test.go

```diff
@@ -164,3 +165,49 @@ func TestFileSystem_Install_RollbackMissingRuntime(t *testing.T) {
 	mixinDirExists, _ := p.FileSystem.DirExists(mixinDir)
 	assert.False(t, mixinDirExists)
 }
+
+func TestFileSystem_Install_MixinInfoSavedWhenNoFileExists(t *testing.T) {
+
+	c := config.NewTestConfig(t)
+	c.SetupPorterHome()
+	p := NewFileSystem(c.Config)
+
+	mixinURL := "https://cdn.deislabs.io/porter/mixins/helm"
```

Comment:
The only change that should be made is to switch from downloading a real mixin to downloading a fake test mixin, because the size is much smaller. It helps our unit tests run _much_ faster.

Here's how the other test does this:

https://github.com/deislabs/porter/blob/bb837ade09a5624ca1bc9aa0fbf69bc7d28293d2/pkg/mixin/provider/install_test.go#L19-L32

Reply:
Thanks. I am aware of the fake test mixin, but did not use it since I did not trigger the installation in the unit test (no `p.Install(opts)`) - its just `p.saveInfo(opts)`. Do you think this is ok?

Reply:
Oh! Silly me yes you are right. It isn't being downloaded so it is fine to use a real URL. 👍